### PR TITLE
💄 Add main button & button variations

### PR DESCRIPTION
**Pull request by ashby-git**
I removed some of the irrelevant styling for the circular button.
Also added, main circular button base and 3 specific buttons.

### CODE REVIEW

**Review by DanArcherOnline — CHANGES_REQUESTED** [review]
the button has only been recreated so its not a refactor. please change the title to reflect the commits in this PR

File: src/components/button/circular-button.styles.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.button {
+  background-color: #555555;
+  border: none;
+  color: white;
+  text-align: center;
+
+  padding-top: 5px;
+ 
+  border-radius: 100%;
+  width: 100px;
```
Comment (DanArcherOnline) [inline src/components/button/circular-button.styles.scss:10]: has it been decided that the buttons will be 100px?
if they will be a different size, best to change it now.
if 100px is fine for the final design, no changes necessary.

File: src/components/button/circular-button.styles.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  height:95px;
+
+  background: #555555;
+
+  display: inline-flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: 60px;
+  color: white;
```
Comment (DanArcherOnline) [inline src/components/button/circular-button.styles.scss:20]: you have specified the color twice

File: src/components/button/refresh-circular-button.component.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import CircularButton from './circular-button.component'
+
+import './circular-button.styles.scss';
+
+const RefreshCircularButton = (props) => (
+
+    <CircularButton className='button'>
```
Comment (DanArcherOnline) [inline src/components/button/refresh-circular-button.component.jsx:9]: don't need the className coz its already set in the CircularButton component.
Also might be better to change the name. Coz you dont really say refresh a counter. maybe reset is better. what do you think?

**Review by DanArcherOnline — CHANGES_REQUESTED** [review]

**Comment by ashby-git** [comment]
I have made the changes you requested, please review the updated code

**Review by DanArcherOnline — APPROVED** [review]
LGTM

**Pull request merged**